package.json

```diff
@@ -24,7 +24,7 @@
   "icon": "icon/icon128.png",
   "engines": {
     "npm": ">=7.0.0",
-    "vscode": "^1.52.0"
+    "vscode": "^1.69.2"
```

Comment:
xml publishes pre-releases with ^1.63.0. Not sure if 1.69.2 is supported by Theia. Any reason you need this particular version?

Reply:
> xml publishes pre-releases with ^1.63.0. Not sure if 1.69.2 is supported by Theia. Any reason you need this particular version?

no specific reason and changed to v1.63.0